### ISSUE TITLE
Parse Error : syntax error, unexpected 'extends' (T_EXTENDS), expecting T_STRING on line 12  10 |  11 | class  > 12 | extends Model   13 | {  14 |  15 | use HasFactory;

### ISSUE THREAD

**Issue by vimitali368** [VemtoOrg/vemto2-issues#123]
**Error Message:**
Parse Error : syntax error, unexpected 'extends' (T_EXTENDS), expecting T_STRING on line 12
  10 |
  11 | class  
> 12 | extends Model 
  13 | {
  14 |
  15 | use HasFactory;

**Stack:**
```
SyntaxError: Parse Error : syntax error, unexpected 'extends' (T_EXTENDS), expecting T_STRING on line 12
  10 |
  11 | class  
> 12 | extends Model 
  13 | {
  14 |
  15 | use HasFactory;
    at mt.raiseError (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:284:175)
    at mt.error (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:284:830)
    at mt.expect (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:285:414)
    at mt.read_class_declaration_statement (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:273:4184)
    at mt.read_top_statement (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:280:1244)
    at mt.read_top_statements (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:280:907)
    at mt.read_namespace (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:273:36852)
    at mt.read_start (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:273:36534)
    at mt.parse (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:283:15386)
    at td.parseCode (file:///C:/Users/w11.vemto/AppData/Local/Programs/Vemto2/resources/app.asar/renderer/assets/index.1c142fe5.js:286:2410)
```

---

Vemto 2.0.10
win32 10.0.25262
Locale: ru
App version: 2.0.10

**Comment by TiagoSilvaPereira** [comment]
Hi @vimitali368 is this a new project or an existing one? Seems some model on your project is missing the parent class:

![image](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/11933789/13d7a2d7-3148-4f7d-aba5-fa5b6b111c34)

**Comment by vimitali368** [comment]
This is a new project.
There are parents models.
![image](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/99253756/94aee0c5-5a59-4cc1-9768-b9662fe3128a)
![image](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/99253756/e4992c34-0be4-4929-92ec-e33f3613f9e0)

**Comment by vimitali368** [comment]
![image](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/99253756/9b2f0b79-ca33-4032-bb96-c7c5f1a9ebda)

**Comment by TiagoSilvaPereira** [comment]
@vimitali368 please check if this post relates to your issue: https://stackoverflow.com/questions/30631635/laravel-5-syntax-error-unexpected-extends-t-extends

Also, does this problem happen when you try to run your application from command line?

**Comment by vimitali368** [comment]
No, it's not related to that.
The error seems to occur when you assign a many-to-many relationship and select the With pivot columns checkbox

**Comment by TiagoSilvaPereira** [comment]
@vimitali368 got it. Would it be possible for you to reproduce the error and record a video so I can reproduce it here?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.11. @vimitali368  please check if you have a model without a name on your Schema definition
